projects/laji-ui/src/lib/typeahead/positioning/positioning.service.ts

```diff
@@ -86,8 +92,8 @@ export class PositioningService {
     }
   }
 
-  position(options: PositioningOptions): void {
-    this.addPositionElement(options);
+  position(options: PositioningOptions, renderer: Renderer2): void {
```

Comment:
Is there a need to pass the renderer as a parameter or could it use the `rendererFactory` like before. Like this for example:
```
private renderer: Renderer2;
constructor(...) {
    ...
    this.renderer = rendererFactory.createRenderer(null, null);
}
```

Reply:
I know passing the renderer as a parameter makes the code a bit more cumbersome to write, but I don't think new renderers should be created unless absolutely necessary (which it isn't in this case).